flink-streaming-java/src/test/java/org/apache/flink/streaming/util/KeyedOneInputStreamOperatorTestHarness.java

```diff
@@ -78,6 +82,20 @@ public int numKeyedStateEntries() {
 		}
 	}
 
+	public <S extends State> S getState(K key, StateDescriptor<S, ?> stateDesc) throws Exception {
```

Comment:
Is this change necessary? We should only modify code outside of `flink-table` if it is urgently needed.